plugins/chat/spec/system/chat_message_creator_spec.rb

```diff
@@ -111,15 +111,16 @@
     user_1 = Fabricate(:user)
     user_2 = Fabricate(:user)
     user_3 = Fabricate(:user)
-    group = Fabricate(:public_group, users: [user_1, user_2])
+    user_4 = Fabricate(:user)
+    group = Fabricate(:public_group, users: [user_1, user_2, user_3])
```

Comment:
Since max dm users is set to 3 in the before block, we should expect adding 4 users to be restricted.